index.html

```diff
@@ -0,0 +1,931 @@
+                       <a href="#" class="greyScale smallLeftMargin noTextDecoration">  David Canellis </a>
+                       <img class="headerIcons" src="Images/clock.png" alt="Clock icon"/>
+                       <span class="greyScale smallMarginLeft noStyle">1 hour ago</span>
+                       <img class="headerIcons" src="Images/share.png" alt="Share icon"/>
+                       <span class="greyScale smallMarginLeft noStyle">4</span>
+                    </div>
+                </div>
+                
+            </div>
+            <article class="article2">
+                <div class="bottomLeft">
+                    <span class="colorWhite">INSIGHTS</span>
+                    <br/>
+                    <h2>
+                    <a href="#" class="descriptionText colorWhite noTextDecoration">
+                        Satoshi Nakaboto: ‘New Visa-powered credit card pays rewards in Bitcoin’
+                    </a>
+                    </h2>
+                    <div class="author">
+                       <span class="greyScale noStyle"> by</span> 
+                       <a href="#" class="greyScale smallLeftMargin noTextDecoration">  Satoshi Nakaboto
+                    </a>
+                       <img class="headerIcons" src="Images/clock.png" alt="Clock Icon"/>                           
+                       <span class="greyScale smallMarginLeft noStyle">10 hour ago</span>
+                        <img class="headerIcons" src="Images/share.png" alt="Share Icon"/>
+                        <span class="greyScale smallMarginLeft noStyle">8</span>
+                    </div>
+                </div>    
+            </article>
+                
+            <article class="article3">
+                <div class="bottomLeft">
+                    <span class="colorWhite">INSIGHTS</span>
+                    <br/>
+                    <h2>
+                    <a href="#" class="descriptionText colorWhite noTextDecoration">
+                        How startups should navigate the business travel shutdown
+                    </a>
+                    </h2>
+                    <div class="author">
+                       <span class="greyScale noStyle"> by</span> 
+                       <a href="#" class="greyScale smallLeftMargin noTextDecoration">  Avi Mier</a>
+                       <img class="headerIcons" src="Images/clock.png" alt="Clock Icon"/>                           
+                       <span class="greyScale smallMarginLeft noStyle">1 hour ago</span>
+                       <img class="headerIcons" src="Images/share.png" alt="Share Icon"/>
+                       <span class="greyScale smallMarginLeft noStyle">12</span>
+                    </div>
+                </div>
+            </article>
+        </section>
+        <section class="latestNews">
+            <div class="latestNewsHeading redHeading"><h2>Latest news > </h2> </div>
```

Comment:
The section is too wide compared to the original. That makes all the elements have larger dimensions. Please make the necessary adjustments.
Check the title of the latest news, too. The font-size is larger in your page.
Yours:
![Screenshot from 2020-04-13 13-26-29](https://user-images.githubusercontent.com/51926937/79148057-87787a00-7d8a-11ea-9b80-5a22e755201e.png)

Original:
![Screenshot from 2020-04-13 13-26-37](https://user-images.githubusercontent.com/51926937/79148072-8cd5c480-7d8a-11ea-83fb-e305242e5b1e.png)